gpjax/parameters/bijectors.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+
+__all__ = ["Bijector", "Identity", "Softplus"]
+
+from dataclasses import dataclass
+from typing import Callable
+
+import jax.numpy as jnp
+from simple_pytree import Pytree, static_field
+
+
+@dataclass
+class Bijector(Pytree):
+    """
+    Create a bijector.
+
+    Args:
+        forward(Callable): The forward transformation.
+        inverse(Callable): The inverse transformation.
+
+    Returns:
+        Bijector: A bijector.
+    """
+
+    forward: Callable = static_field()
+    inverse: Callable = static_field()
+
+
+Identity = Bijector(forward=lambda x: x, inverse=lambda x: x)
+
+Softplus = Bijector(
+    forward=lambda x: jnp.log(1.0 + jnp.exp(x)),
+    inverse=lambda x: jnp.log(jnp.exp(x) - 1.0),
```

Comment:
These are not very numerically stable (and the forward will easily overflow). Better would be
```suggestion
    forward=jax.nn.softplus,
    inverse=tfp.math.softplus_inverse,
```
or, would it be an option to simply use tfp's bijector implementations directly?

Reply:
Nice, thanks @st-- 

Yeah, I favour using TFP. We will be dropping distrax in favour of this too.

(This is just legacy code from Mytree that aimed to be backend dependancy free).